openstack/sharedfilesystems/v2/shares/requests.go

```diff
@@ -379,3 +379,89 @@ func Update(client *gophercloud.ServiceClient, id string, opts UpdateOptsBuilder
 	})
 	return
 }
+
+// ShowMetadata retrieves metadata of the specified share. To extract the retrieved
+// metadata from the response, call the Extract method on the ShowMetadataResult.
+func ShowMetadata(client *gophercloud.ServiceClient, id string) (r ShowMetadataResult) {
```

Comment:
It looks like the Manila code has support to retrieve a single metadata/metadatum: https://github.com/openstack/manila/blob/cec13b8057fa025fa1b1c3bef180687a394d6fd6/manila/api/v1/share_metadata.py#L121


To prevent confusion, let's change this to either `Metadata` or `GetMetadata`. Both have been used in Gophercloud before.

Reply:
+1, this can be GetMetadatum, perhaps?

Reply:
If that API call is exposed, I agree. The API documentation doesn't have it listed, but that might just mean it's undocumented :)

Reply:
Yep! fixing the docs here: https://review.opendev.org/#/c/673112/

Reply:
refactored `ShowMetadata` into `GetMetadata`: https://github.com/gophercloud/gophercloud/pull/1656/commits/da281eda3863bfe81db0c8b0160b0abaf307aced

added `GetMetadatum`: https://github.com/gophercloud/gophercloud/pull/1656/commits/66bd2c0febfe8a1c1f4a02d3f8080739d3c66046